ivy/functional/backends/paddle/data_type.py

```diff
@@ -138,6 +140,17 @@ def broadcast_arrays(*arrays: paddle.Tensor) -> List[paddle.Tensor]:
     return result
 
 
+@with_unsupported_dtypes(
+    {
+        "2.5.1 and below": (
+            "uint8",
+            "int8",
+            "int16",
+            "bfloat16",
```

Comment:
`float16` is also unsupported by the function

Reply:
that's right